net/macos-fortress/Portfile

```diff
@@ -0,0 +1,817 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                macos-fortress
+version             2019.10.27
+revision            0
+
+categories          net security
+platforms           darwin
+license             MIT
+maintainers         {ieee.org:s.t.smith @essandess} openmaintainer
+
+if {${subport} ne "${name}-easylistpac"} {
+    master_sites
+    distfiles
+    extract.only
+}
+
+use_configure       no
+build               {}
```

Comment:
So is this downloading a binary or a shell script?

Reply:
Neither. The entire repo is a configuration of existing MacPorts and macOS tools. The repo’s she’ll script installs everything, but this port doesn’t need that because it does the same thing—better—with this Portfile.